retrieverdash/dashboard_script/sample_dataset.json

```diff
@@ -0,0 +1,171 @@
+{
+    "retriever_minimum_version":"2.0.dev",
+    "keywords":[
+        "mammals",
+        "desert",
+        "time-series",
+        "experimental",
+        "observational"
+    ],
+    "citation":"S. K. Morgan Ernest, Thomas J. Valone, and James H. Brown. 2009. Long-term monitoring and experimental manipulation of a Chihuahuan Desert ecosystem near Portal, Arizona, USA. Ecology 90:1708.",
+    "version":"1.2.1",
+    "title":"Portal Project Data (Ernest et al. 2009)",
+    "resources":[
+        {
+            "dialect":{
+                "delimiter":",",
+                "missingValues":[
+                    ""
+                ],
+                "contains_pk":"True"
+            },
+            "url":"https://raw.githubusercontent.com/apoorvaeternity/sample-dataset/master/original/Portal_rodents_19772002.csv",
+            "name":"main",
```

Comment:
We shall have to add the data first so that the url is from master instead of what you used.
https://raw.githubusercontent.com/apoorvaeternity/sample-dataset/master/original/Portal_rodents_19772002.csv 
This data should be in a data directory somewhere, most probably in the test directory.

Reply:
Okay I will send a PR to add this data in the test directory.

Reply:
The data is up on master